views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -80,7 +80,11 @@
 {% endset %}
 
 {% if pageParams.isRegistration %}
-  {% set changeLinkUrl = OE_CONFIGS.BENEFICIAL_OWNER_GOV_URL + "/" + bog.id %}
+  {% if not IS_REDIS_REMOVAL_ENABLED %}
+    {% set changeLinkUrl = OE_CONFIGS.BENEFICIAL_OWNER_GOV_URL + "/" + bog.id %}
+  {% else %}
+    {% set changeLinkUrl = CREATE_CHANGE_LINK_WITH_IDs(OE_CONFIGS.BENEFICIAL_OWNER_GOV_WITH_PARAMS_URL, OE_TRANSACTION_ID, OE_SUBMISSION_ID) + "/" + bog.id %}
```

Comment:
So looks like URL here could be `#/<<a-bog-id>>`. Is that actually valid as a change link?

Reply:
No, it's not a valid link. This would only avoid the journey from breaking so the user can still see a summary of the changes they made without terminating the journey by loading the error screen.

This is a bit of defensive coding as this scenario is very unlikely to happen. But if it does, we can still show the user a summary of their changes.

Reply:
Yep, I've responded to the previous comment related to this.

Reply:
Thanks. We'll use the related comment as a proxy for this one.